local/db/activity_program.sql

```diff
@@ -0,0 +1,33 @@
+drop table IF EXISTS activity_program;
+
+create table activity_program (
```

Comment:
gormの仕様上
DB名は構造体の複数形という形にしてください。

```suggestion
create table activity_programs (
```